project/frontend/src/components/LocationFinder.js

```diff
@@ -1,5 +1,8 @@
+import './LocationFinder.css';
+
 import React from 'react';
-import MyLocation from '@material-ui/icons/MyLocation';
+import Loading from './Loading.js';
+import MyLocation from '../assets/gps.svg';
```

Comment:
Does this mean that you just saved the MyLocation icon locally in our project?

Reply:
Yeah, I don't remember if its the same icon, but I wanted more control over the styling and stuff.